src/test/java/seedu/address/logic/commands/AddCommandTest.java

```diff
@@ -113,6 +114,16 @@ public void addPerson(Person person) {
             throw new AssertionError("This method should not be called.");
         }
 
+        @Override
+        public void addDoctor(Doctor doctor) {
```

Comment:
May I clarify if the assertion error is a placeholder for now?

Reply:
Yes, it is a placeholder for now. Test cases will be written for AddDoctor command once the initial set of functions is complete.